app/angular/src/client/preview/angular/helpers.ts

```diff
@@ -17,6 +17,8 @@ type DynamicComponentType = typeof componentClass;
 
 const storyData = new ReplaySubject(1);
 
+declare var NODE_ENV: string | 'development';
```

Comment:
Is this supposed to set the default value to `'development'` if NODE_ENV isn't defined? I don't think it does

Reply:
This specifies the possible value.